File: week6python/pset6/cash/cash.py
import cs50

change = cs50.get_float("Change: ")
while change < 0:
    change = cs50.get_float("Change owed: ")
change = round(change * 100)

coins = 0
while change >= 25:
    change -= 25
    coins += 1
while change >= 10:
    change -= 10
    coins += 1
while change >= 5:
    change -= 5
    coins += 1
while change >= 1:
    change -= 1
    coins += 1
print(coins)

File: week1/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)

{
    //prompt input for change
    float change;
    do
    {
        change = get_float("change owed:");
    }
    while (change <= 0);
    
    //convert dollars to cents
    int cents;
    cents = round (change * 100);
    
    //counting number coins needed
    
    //number of quarters (25cents)
    int coins = 0;
    while (cents >= 25)
    {
        cents = cents - 25;
        coins++;
    }
    
    //number of dimes (10cents)
    while (cents >= 10)
    {
        cents = cents - 10;
        coins++;
    }
    
    //number of nickels (5cents)
    while (cents >= 5)
    {
        cents = cents - 5;
        coins++;
    }
    
    //number of pennies (1cent)
    while (cents >= 1)
    {
        cents = cents - 1;
        coins++;
    }
    
    printf("%i\n", coins);
}